src/js/main/index.js

```diff
@@ -355,10 +356,31 @@ $(document).ready(function(){
         },
         minLength: 2,
         select: function(event, ui){
-            if(ga){
-                ga('send', 'event', 'LANDING_PAGE', 'company-query-autocomplete-select', ui['item']['company_id']);
-            }
-            $("#company_id").val(ui['item']['company_id']);
-        }
+            $company_query.trigger('autocomplete-select', ui.item.company_id);
+            $("#company_id").val(ui.item.company_id);
```

Comment:
seems not only typo, but also the id is changed

`#company_id` --> `#form-company-id`